include/libopencm3/sam/4l/pm.h

```diff
@@ -0,0 +1,184 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2016 Maxim Sloyko <maxim@sloyko.com>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_PM_H
+#define LIBOPENCM3_PM_H
+
+#include <libopencm3/sam/memorymap.h>
+
+/* --- Power Manager (PM) registers ----------------------- */
+
+/* Main Clock Control MCCTRL Read/Write */
+#define PM_MCCTRL			MMIO32(PM_BASE + 0x000)
+
+/* CPU Clock Select CPUSEL Read/Write */
+#define PM_CPUSEL			MMIO32(PM_BASE + 0x004)
+
+/* PBA Clock Select PBASEL Read/Write */
+#define PM_PBASEL			MMIO32(PM_BASE + 0x00C)
+
+/* PBB Clock Select PBBSEL Read/Write */
+
+#define PM_PBBSEL			MMIO32(PM_BASE + 0x010)
+/* PBC Clock Select PBCSEL Read/Write */
```

Comment:
you get out of sync with your comments here, you end up with an extra line between things